.travis.yml

```diff
@@ -40,3 +48,31 @@ script:
   - curl -k https://localhost:444
   - curl -k https://localhost:8443
   - curl -k https://localhost:8444
+
+after_success:
+  - curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+  - chmod +x ./kubectl
+  - sudo mv ./kubectl /usr/local/bin/kubectl
+  - echo -n ${travis} | base64 -d > travis-service-account.json
+  - gcloud auth activate-service-account travis@api-platform-demo-209807.iam.gserviceaccount.com --key-file travis-service-account.json --project=api-platform-demo-209807
+  - gcloud config set compute/zone europe-west3-c
+  - gcloud config set project api-platform-demo-209807
+  - tar -zxvf helm-v2.7.2-linux-amd64.tar.gz
```

Comment:
this is already done here https://github.com/api-platform/demo/pull/26/files#diff-354f30a63fb0907d4ad57269548329e3R22